otx/cli/utils/tests.py

```diff
@@ -84,6 +84,37 @@ def otx_train_testing(template, root, otx_dir, args):
     assert os.path.exists(f"{template_work_dir}/trained_{template.model_template_id}/label_schema.json")
 
 
+def otx_resume_testing(template, root, otx_dir, args):
+    template_work_dir = get_template_dir(template, root)
+    command_line = [
+        "otx",
+        "train",
+        template.model_template_path,
+    ]
+    for option in [
+        "--data",
+        "--train-ann-file",
+        "--train-data-roots",
+        "--val-ann-file",
+        "--val-data-roots",
+        "--unlabeled-data-roots",
+        "--unlabeled-file-list",
+        "--resume-from",
+    ]:
+        if option in args:
+            command_line.extend([option, f"{os.path.join(otx_dir, args[option])}"])
+
+    command_line.extend(["--save-model-to", f"{template_work_dir}/trained_for_resume_{template.model_template_id}"])
+    if "--gpus" in args:
```

Comment:
Should multi-gpu be considered for resume testing?

Reply:
We also do not have any reason not to consider multi-gpu.

Reply:
Fixed.